lfucg/lexingtonky.gov
# Adds scheduling to all content types

## Summary / Approach

Adds the Scheduler module and sets up the ability to schedule on ALL content types. But, there's a catch. Workbench Moderation and Scheduler do not play nicely with each other so I had to write some hooks to manually toggle the state of the content when Scheduler tries to work its magic. 

## Testing Instruction(s)

Schedule two items for publish/unpublish and wait until the scheduled time and refresh to see them toggle. /

This relies on cron. There's a new cron job to run every minute for scheduler. Luckily it's low risk to run that often. 

## Screenshots

None really. 

## Metadata

| Question | Answer |
|----------|--------|
| Did you perform a self-review of this PR? | yes, unfortunately.
| Documentation reflects changes? | N/A
| Unit/Functional tests reflect changes? | N/a
| Did you perform browser testing? | n/a
| Did you provide detail in the summary on how and where to test this branch? | yes
| Relevant links | https://apaxsoftware.atlassian.net/browse/LG-46
| Scheduler Issue | https://www.drupal.org/project/scheduler/issues/2977887#comment-14541704
